docs/declaring-loaders.rst

```diff
@@ -0,0 +1,38 @@
+.. _declaring-loaders:
+
+Declaring Item Loaders
+======================
+
+Item Loaders are declared by using a class definition syntax. Here is an example::
+
+    from itemloaders import ItemLoader
+    from itemloaders.processors import TakeFirst, MapCompose, Join
+
+    class ProductLoader(ItemLoader):
+
+        default_output_processor = TakeFirst()
+
+        name_in = MapCompose(unicode.title)
```

Comment:
this example doesn't work, because there is no `unicode` in Python 3. It should be str.title. Below is a similar error.